_api-reference/list/index.md

```diff
@@ -0,0 +1,175 @@
+`h`  |  Limits the output to specific headers.
+`format` |  The format in which to return the result. Valid values are `json`, `yaml`, `cbor`, and `smile`.
+`s` | Sorts the output by the specified columns.
+
+## Examples
+
+The following examples show how to use the optional query parameters to customize all List API responses.
+
+
+### Get verbose output
+
+To query indexes and their statistics with a verbose output that includes all column headings in the response, use the `v` query parameter, as shown in the following example.
+
+#### Request
+
+```json
+GET _list/indices?v
+```
+{% include copy-curl.html %}
+
+#### Response
+
+```json
+health status index           uuid    pri rep  docs.count  docs.deleted
+green  open   .kibana_1 - - - -              
+yellow open    sample-index-1 - - - -
+next_token null
+```
+
+
+### Get all available headers
+
+To see all the available headers, use the `help` parameter with the following syntax:
+
+```json
+GET _list/<operation_name>?help
+```
+{% include copy-curl.html %}
+
+#### Request
+
+The following example list indices operation returns all the available headers:
+
+```json
+GET _list/indices?help
+```
+{% include copy-curl.html %}
+
+#### Response
+
+The following example displays the indexes and their health status in a table:
+
+```json
+health     | h                              | current health status
+status     | s                              | open/close status
+index      | i,idx                          | index name
+uuid       | id,uuid                        | index uuid
+pri        | p,shards.primary,shardsPrimary | number of primary shards
+rep        | r,shards.replica,shardsReplica | number of replica shards
+docs.count | dc,docsCount                   | available docs
+```
+
+### Get a subset of headers
+
+To limit the output to a subset of headers, use the `h` parameter with the following syntax:
+
+```json
+GET _list/<operation_name>?h=<header_name_1>,<header_name_2>&v
+```
+{% include copy-curl.html %}
+
+For any operation, you can determine which headers are available by using the `help` parameter and then using the `h` parameter to limit the output to only a subset of headers. 
+
+#### Request
+
+The following example limits the indexes in the response to only the index name and health status headers:
+
+```json
+GET _list/indices?h=health,index
+```
+{% include copy-curl.html %}
+
+### Response
+
+```json
+green  .kibana_1
+yellow sample-index-1
+next_token null
+```
+
+
+### Sort by a header
+
+To sort the output on a single page by a header, use the `s` parameter with the following syntax:
+
+```json
+GET _list/<operation_name>?s=<header_name_1>,<header_name_2>
+```
+{% include copy-curl.html %}
+
+#### Request
+
+The following example request sorts indexes by index name:
+
+```json
+GET _list/indices?s=h,i
+```
+{% include copy-curl.html %}
+
+#### Response
+
+```json
+green sample-index-2
+yellow sample-index-1
+next_token null
+```
+
+### Retrieve data in JSON format
```

Comment:
Can we add these examples in individual list-shards and list-indices pages?

Reply:
I think it makes more sense to keep these on the main page, since it these example pertain specifically to Query parameters supported by both endpoints, and means less examples to maintain on the individual pages. However, I can add a link to these examples on both pages.

Reply:
On this, the only thing is the Key of JSON response changes when moving from _list/indices to _list/shards. Basically, the response would look like:

```
_list/indices

```
```
{"next_token": "", "indices": ["", ""]}
```

```

_list/shards

```

```
{"next_token": "", "shards": ["", ""]}

```

I think the comment from @sumitasr makes sense, and I can revise the PR addressing it. @Naarcha-AWS Would it be okay?

Reply:
Go ahead.

Reply:
@sumitasr Added. Please check.